packages/portal/src/components/search/SearchInterface.vue

```diff
@@ -25,115 +25,89 @@
             />
           </transition>
         </client-only>
-        <b-container
-          class="px-0 pb-3"
-        >
-          <section>
-            <div
-              class="mb-3 d-flex flex-wrap align-items-center justify-content-between"
+        <LoadingSpinner
+          v-show="$fetchState.pending"
+          class="flex-md-row py-4 text-center"
+          :status-message="$t('loadingResults')"
+        />
+        <template v-if="!$fetchState.pending">
+          <ErrorMessage
+            v-if="$fetchState.error"
+            :error="$fetchState.error"
+            :full-height="false"
+            :show-message="showErrorMessage"
+            title-tag="h2"
+          />
+          <b-row v-else>
+            <!-- TODO: check this is working as intended after the ItemPreviewInterface refactor -->
+            <p
+              v-show="noMoreResults"
+              data-qa="warning notice"
             >
-              <!-- This div prevents SearchViewToggles jumping around as SearchResultsContext is shown & hidden -->
-              <div v-show="$fetchState.pending" />
-              <SearchResultsContext
-                v-show="!$fetchState.pending"
-                :total-results="totalResults"
-                :entity="$store.state.entity.entity"
-                :query="query"
-                badge-variant="primary-light"
-              />
-              <SearchViewToggles
-                v-model="view"
-                class="ml-auto"
-              />
-            </div>
-            <b-row
-              class="mb-3"
+              {{ $t('noMoreResults') }}
+            </p>
+            <ItemPreviewInterface
+              data-qa="liked items"
+              :items="results"
+              :hits="hits"
+              :max-results="1000"
+              :per-page="perPage"
+              :show-pins="showPins"
+              :total="totalResults"
+              :on-aux-click-card="onClickItem"
+              :on-click-card="onClickItem"
+              class="w-100 mb-3"
+              @drawn="handleResultsDrawn"
             >
-              <b-col
-                cols="12"
+              <template #heading>
+                <SearchResultsContext
+                  :total-results="totalResults"
+                  :entity="$store.state.entity.entity"
+                  :query="query"
+                  badge-variant="primary-light"
+                />
+              </template>
+              <template
+                v-if="noResults"
+                #no-items
+              >
+                <ErrorMessage
+                  :error="{ code: 'searchResultsNotFound', i18n: $t('errorMessage.searchResultsNotFound') }"
+                  :full-height="false"
+                  :show-message="false"
+                  title-tag="h2"
+                />
+              </template>
+              <template #card-group-default>
```

Comment:
This slot is used by collection pages to insert the EntityHeader. So now we are naming the slot we might want to call it something more relevant like `collection-header-card`.

Reply:
I have renamed slots to make explicit that they are for the card group, and this particular one becomes card-group-header all the way down then finally header.

Reply:
Great. Only missed the entity header slot for the list view in ItemPreviewCardGroup. I've pushed a commit to make that so.